packages/import-mistica-icons/index.js

```diff
@@ -194,6 +198,35 @@ const createIconComponentSource = async (name, componentName, svgIconsInfo) => {
     return format(source);
 };
 
+/**
+ * @typedef {{[key: string]: Array<string>}} iconKeywordsInfo
+ */
+const createAllIconKeywordsSource = () => {
+    const keywordsPath = join(PATH_MISTICA_ICONS_REPO, 'icons', 'icons-keywords.json');
+    const keywordsSource = fs.readFileSync(keywordsPath, 'utf8');
+    /** @type {iconKeywordsInfo} */
+    const keywordsMap = JSON.parse(keywordsSource);
+
+    // map icon names to kebab case
```

Comment:
some icon filenames in Mistica design repo are not following the expected format when using kebab case. For example: `2g-filled` instead of `2-g-filled`.

Given that in Mistica web we generate the filenames using kebab case (check the name of [this file](https://github.com/Telefonica/mistica-web/blob/master/src/generated/mistica-icons/icon-2-g-filled.tsx) for example), I mapped the names in design repo to their corresponding kebab case version when generating the keywords file.